src/main/java/com/project/model/User.java

```diff
@@ -1,20 +1,25 @@
 package com.project.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.security.core.authority.SimpleGrantedAuthority;
 import org.springframework.security.core.userdetails.UserDetails;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
+
+import java.util.*;
```

Comment:
You should not import everything from a package if you do not need it. Better to just import what you need.

Reply:
Intellij automatically changes imports to import package.* when there are more than 4 elements imported from the same package. As there are some classes with more than 8 or 10 imports from the same package, i think it is better to let intellij handle the imports as writing them manually every time you add a new class to import from the same package is going to be time consuming